test-support/build.gradle

```diff
@@ -0,0 +1,67 @@
+import org.gradle.util.GradleVersion
+
+plugins {
+    id 'java-library'
+    id 'maven-publish'
```

Comment:
What do we need the maven publishing for?

Reply:
I think it is fine to publish this as it might be useful to us or others. But I guess that also implies that it is "supported" and reduces our flexibility in making changes.

We can remove the publishing and add it later if we really need it.

Reply:
Fixed and force-pushed.